fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -54,6 +58,7 @@ import org.mozilla.fenix.theme.FirefoxTheme
 fun TranslationsDialogBottomSheet(
     learnMoreUrl: String,
     showFirstTimeTranslation: Boolean,
+    translationError: TranslationError? = null,
```

Comment:
Talking this through for myself, so, an AC call to `TranslateUseCase` will occur, we will then have middleware that will pick-up an error occurred, and then the middleware will evoke the request to show an error.

I think that makes sense to me, is that what you are thinking too? 

Also, pinging @gabrielluong for thoughts too.

Reply:
Assuming we are observing this state, then yes this makes sense.